test/perf/perf_sort_by_proximity.cc

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+ */
+
+#include <seastar/core/reactor.hh>
+#include <seastar/core/app-template.hh>
+#include <seastar/core/sstring.hh>
+#include <seastar/core/sleep.hh>
+#include <seastar/core/thread.hh>
+#include <seastar/rpc/rpc_types.hh>
+#include "inet_address_vectors.hh"
+#include "seastarx.hh"
+
+#include <seastar/testing/perf_tests.hh>
+#include <seastar/testing/test_runner.hh>
+
+#include "locator/token_metadata.hh"
+
+struct sort_by_proximity_topology {
+    static constexpr size_t DCS = 1;
+    static constexpr size_t RACKS_PER_DC = 3;
+    static constexpr size_t NODES_PER_RACK = 1;
```

Comment:
It would be more realistic to have a larger NODES_PER_RACK, so that we don't touch exactly the same nodes each time.

Reply:
Done in v6 ([d1490bb](https://github.com/scylladb/scylladb/commit/d1490bb7bfcdabe36ff50d88379e7edb7cbe2ddf))